### ISSUE TITLE
File system sync directory change doesn't work  (Linux desktop)

### ISSUE THREAD

**Issue by technicalguy** [laurent22/joplin#536]
### Operating system
- Linux

### Application
- Desktop

Steps to reproduce:

1. Install joplin fresh (with no `~/.config/joplin` or `~/.config/joplin-desktop`)
0.  - Synchronisation target: File system
    - Directory to synchronise with: `/home/rms/Documents/myfirstsyncfolder`
0. Create some notes and sync them
0. Delete folder `/home/rms/Documents/myfirstsyncfolder`
0. Create folder `/home/rms/Pictures/mysecondsyncfolder`
0. Update Joplin settings: Directory to synchronise with: `/home/rms/Pictures/mysecondsyncfolder`
0. Sync Joplin

### Expected behaviour:  
Joplin syncs to `/home/rms/Pictures/mysecondsyncfolder`

### Observed behaviour:
Joplin doesn't update the sync folder and keeps syncing to `/home/rms/Documents/myfirstsyncfolder` even after it was deleted.

### Workaround:

1. Backup your notes
0. Delete `~/.config/joplin` and `~/.config/joplin-desktop`
0. Start Joplin and setup with new sync folder

**Comment by laurent22** [comment]
Does it really keep syncing with myfirstsyncfolder? Does it create new notes inside, etc?

The correct way to change the sync folder is like this:

1. Copy `/home/rms/Documents/myfirstsyncfolder` to `/home/rms/Documents/mysecondsyncfolder`
2. In Joplin, change dir to `/home/rms/Documents/mysecondsyncfolder`
3. Delete `/home/rms/Documents/myfirstsyncfolder`

**Comment by laurent22** [comment]
Indeed something doesn't look right - marking it as bug.

**Comment by technicalguy** [comment]
It's working great now on 1.0.94!

**Comment by m160** [comment]
I'm still having a problem with this. I copied .config/joplin-desktop to Documents/joplin, went into the the desktop app and made **synchronisation target**: _File system_ and
**directory to synchronise with**: _/home/michael/Documents/joplin_

When I synced, the app deleted all the notes.

I had all the notes backed up so it wasn't a problem. My solution was to just symlink both .config/joplin-desktop and .config/joplin to Documents/joplin so both the desktop and terminal app stays in sync.

Is this okay, or will it cause issues if running both the apps at the same time etc?
